### DISCUSSION TITLE
Update function not used anymore in conv?

### DISCUSSION THREAD

**Discussion by Jeriousman** [pyg-team/pytorch_geometric#4005] [Q&A] [answered]
I have been checking PyG built-in conv layers like GATConv, SAGEConv, and GIN_CONV and seen that update function was not defined to update embeddings anywhere. Why is that? Is the update function fused in somewhere? Can you please clarify this?

**Comment by rusty1s** [comment, marked as answer]
The `update` method is still supported. However, in our code base, we mostly removed it and instead placed all of its original code simply after the `propagate` call. This has the same effect but is probably easier to read and understand.

**Reply by Jeriousman** [reply]
I see. Thank you. Unless I investigate source code thoroughly,  just with documentation it would be a bit hard to pin I guess. Thank you for your answer
